Wrappers/Python/cil/utilities/display.py

```diff
@@ -497,7 +497,10 @@ def __show2D(self,datacontainers, title=None, slice_list=None, fix_range=False,
                             cut_axis[1] = data.dimension_labels[cut_axis[1]]
 
                         temp_dict = {cut_axis[0]:cut_slices[0], cut_axis[1]:cut_slices[1]}
-                        plot_data = data.get_slice(**temp_dict, force=True)
+                        if type(data) == DataContainer:
+                            plot_data = data.get_slice(**temp_dict)
```

Comment:
Instead, update get_slice in DataContainer to dismiss 'force' parameter. i.e. remove from kwargs before continuing